src/packages/@library/icons/FolderPlus.svelte

```diff
@@ -0,0 +1,10 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_3439_4638)">
+<path d="M13.3327 11.3333C13.3327 12.0687 12.7348 12.6667 11.9993 12.6667H3.99935C3.26393 12.6667 2.66602 12.0687 2.66602 11.3333V4.66666C2.66602 3.93124 3.26393 3.33333 3.99935 3.33333H6.66602C7.08477 3.33333 7.4806 3.53124 7.73268 3.86666L8.13268 4.39999C8.25768 4.56874 8.4556 4.66666 8.66602 4.66666H11.9993C12.7348 4.66666 13.3327 5.26458 13.3327 5.99999V11.3333ZM7.49935 10.5C7.49935 10.7771 7.72227 11 7.99935 11C8.27643 11 8.49935 10.7771 8.49935 10.5V9.16666H9.83268C10.1098 9.16666 10.3327 8.94375 10.3327 8.66666C10.3327 8.38958 10.1098 8.16666 9.83268 8.16666H8.49935V6.83333C8.49935 6.55625 8.27643 6.33333 7.99935 6.33333C7.72227 6.33333 7.49935 6.55625 7.49935 6.83333V8.16666H6.16602C5.88893 8.16666 5.66602 8.38958 5.66602 8.66666C5.66602 8.94375 5.88893 9.16666 6.16602 9.16666H7.49935V10.5Z" fill="#8A9299"/>
```

Comment:
<script>
    export let height = "24px";
    export let width = "24px";
    export let color = "grey";
  </script>
  
add this script, and pass value as a props